sdk/lib/_http/http_impl.dart

```diff
@@ -2151,6 +2152,13 @@ class _HttpClient implements HttpClient {
 
   _HttpClient(this._context);
 
+  static bool get _isHttpAllowedByDefault {
+    if (Platform.isIOS) return true;
```

Comment:
Maybe opposite value, i.e. `false`?

Reply:
That would be a breaking change upstream.

I would be happy to do it but @mit-mit needs to agree.